pilot/pkg/kube/inject/webhook.go

```diff
@@ -188,12 +188,18 @@ func NewWebhook(p WebhookParameters) (*Webhook, error) {
 // Run implements the webhook server
 func (wh *Webhook) Run(stop <-chan struct{}) {
 	go func() {
-		if err := wh.server.ListenAndServeTLS("", ""); err != nil {
-			log.Errorf("ListenAndServeTLS for admission webhook returned error: %v", err)
+		err := wh.server.ListenAndServeTLS("", "")
+
+		msg := fmt.Sprintf("Stopped listening on %s", wh.server.Addr)
+		select {
+		case <-stop:
+			log.Infof(msg)
+		default:
+			log.Fatalf("%s due to error: %v", msg, err)
 		}
 	}()
-	defer wh.watcher.Close() // nolint: errcheck
-	defer wh.server.Close()  // nolint: errcheck
+	defer wh.watcher.Close()
+	defer wh.server.Close()
```

Comment:
Does this no longer cause a linter error?

Reply:
No, it doesn't now.